silkworm/rpc/ws/connection.hpp

```diff
@@ -35,6 +36,9 @@
 
 namespace silkworm::rpc::ws {
 
+//! The default ping interval
+constexpr std::chrono::milliseconds kDefaultPingInterval{60'000};
```

Comment:
You can move this constant to `connection.cpp` so that you avoid recompilation of user code when it changes.